mod/reportedcontent/views/default/reportedcontent/css.php

```diff
@@ -4,3 +4,6 @@
 .elgg-icon-report-this:before {
 	content: "\f071";
 }
+.elgg-menu-extras .elgg-menu-item-report-this .elgg-anchor-label {
+	display: none;
```

Comment:
Inaccessible way to hide text, but with font icons I don't have a better solution.